adapter/application.go

```diff
@@ -0,0 +1,35 @@
+package adapter
+
+import "github.com/go-openapi/strfmt"
+
+// Copied from fn_go/modelsv2/app.go
+// We need to maintain this struct since we have already exposed it to the user
+type App struct {
```

Comment:
Thinking out loud here: Is there a reason we want to copy-paste this all, or could we just consume the existing `models` structs from `fn_go`?

Reply:
We would like to be able to eventually retire the `fn_go` library dependency form this Fn CLI. This change allows us to take a step in that direction.